docs/guides/dsp-guide.md

```diff
@@ -10,6 +10,15 @@ sidebar_position: 05
 
 This guide is for DSPs who transact on UID2s in the bid stream.
 
+DSPs receive UID2 tokens in bid requests, and decrypt the UID2 tokens to arrive at raw UID2s that they can use for bidding, using one of the server-side SDKs that support this function:
+
+- UID2 SDK for Java: see [UID2 SDK for Java (Server-Side) Reference Guide](../sdks/uid2-sdk-ref-java.md)
```

Comment:
This could get out of date when we add languages.
Should we point them to https://unifiedid.com/docs/sdks/summary-sdks instead?

Reply:
Updated, thx.